doc/source/conf.py

```diff
@@ -365,4 +425,4 @@ def setup(app: sphinx.application.Sphinx) -> None:
         Sphinx application instance containing the all the doc build configuration.
     """
 
-    app.connect("builder-inited", copy_download_files_to_source_dir)
+    app.connect("builder-inited", archive_examples)
```

Comment:
Can we add a new hook for removing the examples from the `app.src` once the build has finished? This way we avoid cluttering the doc/source directory.

Reply:
The examples aren't saved to doc/source. They are saved to root/build and that is included in .gitignore.